### PR TITLE
Minor readability fixes in README

### PR DESCRIPTION
**Pull request by rionmonster**
Changed phrasing of "just added" to "recently added".
Updated sentence structure in Getting Started section to add clarity (usage of "either" was in a single sentence so I adjusted to improve readability)

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-csharp?pullRequest=603) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-csharp?pullRequest=603) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-csharp?pullRequest=603) it.</sub>

File: README.md [hunk 1]
```diff
@@ -86,11 +86,11 @@ For value, insert your SendGrid API key. Click "Ok", and once again in the Varia
 
 You may need to restart your IDE to make use of the new variable.
 
-Now, if all went well, you can access the just added variable in your C# SendGrid projects! Ready for some examples?
+Now, if all went well, you can access the newly added variable in your C# SendGrid projects! Ready for some examples?
 
 ## Install Package
 
-To use SendGrid in your C# project, you can either <a href="https://github.com/sendgrid/sendgrid-csharp.git">download the SendGrid C# .NET libraries directly from our Github repository</a>. If you have the NuGet package manager installed, you can grab them automatically:
+To use SendGrid in your C# project, you can either <a href="https://github.com/sendgrid/sendgrid-csharp.git">download the SendGrid C# .NET libraries directly from our Github repository</a>or if you have the NuGet package manager installed, you can grab them automatically:
```
Comment (thinkingserious) [inline README.md]: This change breaks the formatting, a space is needed.

Comment (rionmonster) [reply README.md]: This should be addressed in the latest commit.

**Comment by thinkingserious** [comment]
@rionmonster

We have not been able to merge your Pull Request, but because you are awesome - we wanted to make sure you could still get a SendGrid Hacktoberfest shirt.  

Please go **fill [out our swag form](https://dx.sendgrid.com/swag) before Nov 5th** and we will send the shirt! (We know that you might have tried this before and it didn’t work, sorry about that!)

### **You have till Nov 5th to fill out this form** in order to get the Hacktoberfest shirt!

Thank you for contributing during Hacktoberfest! We hope to see you in the repos soon! Just so you know, we always give away a SendGrid shirt for your first ever non-Hacktoberfest PR that gets merged.

**Comment by thinkingserious** [comment]
Hello @rionmonster,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/announcing-hacktoberfest-2017">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**